packages/core/core/src/AssetGraph.js

```diff
@@ -114,44 +126,45 @@ export default class AssetGraph extends Graph {
   }
 
   // Once a dependency is resolved, connect it to a node representing the file it was resolved to
-  updateDependency(dep: Dependency, file: File): DepUpdates {
-    let newFile;
+  updateDependency(dep: Dependency, req: TransformerRequest): DepUpdates {
+    let newRequest;
 
     let depNode = nodeFromDep(dep);
     this.incompleteNodes.delete(depNode.id);
     this.invalidNodes.delete(depNode.id);
 
-    let fileNode = nodeFromFile(file);
-    let {added, removed} = this.replaceNodesConnectedTo(depNode, [fileNode]);
+    let requestNode = nodeFromTransformerRequest(req);
+    let {added, removed} = this.replaceNodesConnectedTo(depNode, [requestNode]);
 
     if (added.nodes.size) {
-      newFile = file;
-      this.incompleteNodes.set(fileNode.id, fileNode);
+      newRequest = req;
+      this.incompleteNodes.set(requestNode.id, requestNode);
     }
 
     let prunedFiles = getFilesFromGraph(removed);
-    return {newFile, prunedFiles};
+    return {newRequest, prunedFiles};
   }
 
   // Once a file has been transformed, connect it to asset nodes representing the generated assets
-  updateFile(file: File, cacheEntry: CacheEntry): FileUpdates {
+  updateFile(req: TransformerRequest, cacheEntry: CacheEntry): FileUpdates {
```

Comment:
We should probably change the name of this method. I'd be okay with `updateTransformerRequest`, but I'm also thinking that it would be helpful to indicate that we're not just updating something, we are marking it as complete. So maybe something like `completeTranformerRequest`/`completeDependency[Request]` or `resolveTranformerRequest`/`resolveDependency[Request]`?